scripts/obtain/dotnet-install.ps1

```diff
@@ -140,6 +140,17 @@ function GetHTTPResponse([Uri] $Uri)
     try {
         # HttpClient is used vs Invoke-WebRequest in order to support Nano Server which doesn't support the Invoke-WebRequest cmdlet.
         Load-Assembly -Assembly System.Net.Http
+ 
+        if(-not $ProxyAddress)
```

Comment:
I think my comment on https://github.com/dotnet/cli/pull/5865 applies here too: nothing in `dotnet-install.sh` refers to proxy servers and users can use `http_proxy/HTTP_PROXY` environment variables to set the proxy directly. Using `~/.curlrc` is also an option.